frontend/.prettierrc.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  printWidth: 130,
+  printWidth: 80,
```

Comment:
do we really have such small screens? ^^

Reply:
@internetti uses multiple panes alongside one another. I think 80 is also the recommended setting in python linting based on that. Typically I go with 100.